src/components/button/_button.scss

```diff
@@ -12,6 +12,7 @@ $button-border-height: 3px;
   font-family: inherit;
   font-size: 1rem;
   font-weight: 600;
+  line-height: 1.15;
```

Comment:
Took me a while to figure out what this was fixing, so might be worth commenting that this fixes the CTA button ore be more specific with the fix so something like:

```
.btn [class*='icon--'] { 
    line-height: 1.15;
}
```

to specifically show this is a fix for buttons with icons.

Reply:
@bameyrick It doesn't fix the` icon--` class it fixes the `.btn` class as the `line-height` declaration is required for both `button` and `a` implementations. The same `line-height` value is also declared on the `button` element which isn't specific to the `.btn`.